content/applications/essentials/activities.rst

```diff
@@ -2,237 +2,348 @@
+Enter a :guilabel:`Name` for the activity type at the top of the form, then enter the following
+information on the form.
 
-.. note::
-   If an activity has the :guilabel:`Chaining Type` set to :guilabel:`Suggest Next Activity`, and
-   has activities listed in the :guilabel:`Suggest` field, users are presented with recommendations
-   for activities as next steps.
+Activity Settings section
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :guilabel:`Action`: using the drop-down menu, select an action associated with this new activity
+  type. Some actions trigger specific behaviors after an activity is scheduled, such as:
+
+  - :guilabel:`Upload Document`: if selected, a link to upload a document is automatically added to
+    the planned activity in the chatter.
+  - :guilabel:`Call` or :guilabel:`Meeting`: if selected, users have the option to open their
+    calendar to select a date and time for the activity.
+  - :guilabel:`Request Signature`: if selected, a link to open a signature request pop-up window is
+    automatically added to the planned activity in the chatter.
+
+  .. note::
+     Available activity types vary based on the installed applications in the database.
 
-Trigger next activity
+- :guilabel:`Folder`: this field **only** appears if :guilabel:`Upload Document` is selected for the
+  :guilabel:`Action`.
+
+  Using the drop-down menu, select the :guilabel:`Folder` the document is saved to.
+
+- :guilabel:`Default User`: select a user from the drop-down menu to automatically assign this
+  activity to the selected user when this activity type is scheduled. If this field is left blank,
+  the activity is assigned to the user who creates the activity.
+- :guilabel:`Default Summary`: enter a note to include whenever this activity type is created.
+
+  .. note::
+     The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields are
+     included when an activity is created. However, they can be altered before the activity is
+     scheduled or saved.
+
+- :guilabel:`Keep Done`: tick this checkbox to keep activities that have been marked as `Done`
+  visible in the :ref:`activity view <activities/activity>`.
+- :guilabel:`Default Note`: enter any notes to appear with the activity.
+
+Next Activity section
 ~~~~~~~~~~~~~~~~~~~~~
 
-Setting the :guilabel:`Chaining Type` to :guilabel:`Trigger Next Activity` immediately launches the
-next activity once the previous one is completed.
+It is possible to have another activity either suggested or triggered. To do so, configure the
+:guilabel:`Next Activity` section.
 
-If :guilabel:`Trigger Next Activity` is selected in the :guilabel:`Chaining Type` field, the field
-beneath changes to: :guilabel:`Trigger`. From the :guilabel:`Trigger` field drop-down menu, select
-the activity that should be launched once this activity is completed.
+- :guilabel:`Chaining Type`: select either :guilabel:`Suggest Next Activity` or :guilabel:`Trigger
+  Next Activity` from the drop-down menu.
```

Comment:
Suggestion to explain that selecting an option in this field, affects the display of other fields
```suggestion
- :guilabel:`Chaining Type`: select either :guilabel:`Suggest Next Activity` or  :guilabel:`Trigger
  Next Activity` from the drop-down menu. Depending on the selected option, either the
  :guilabel:`Suggest` or :guilabel:`Trigger` field is displayed.
```